docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -11,6 +11,20 @@ endif::[]
 [id="{p}-ga-upgrade"]
 == Upgrade to ECK {eck_version}
 
+[IMPORTANT]
+================================
+This is the upgrade guide for ECK. For Elastic Stack upgrade, please refer to this <<{p}-upgrading-stack,guidance>> for more details.
+================================
+
+[CAUTION]
+================================
+Upgrading the ECK operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods, on which Elastic Stack components are running, by Kubernetes to apply those changes. Therefore, if you have a large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart will cause performance degradation, which might be disruptive or inconvenient.
+
+For production use, when you plan to upgrade ECK, it's important to include this pods rolling restart into time schedule consideration.
+
+Also as described later, you could temporarily exclude the Elastic Stack components from being managed by ECK operator. After the ECK upgrade is done, you must remember to remove the exclusion to ensure that your deployment is continually monitored and managed by the operator, and let the rolling restart go through.
```

Comment:
We are already mentioning this a few paragraphs down. I am a bit worried if we pepper this page with admonition paragraphs that it will not necessary lead to better reader comprehension but instead readers will just start to ignore the many exclamation marks.

Reply:
Thanks! Totally agree with this when thinking about the better reader comprehension! I will remove the duplicated part at the top.